site/content/docs/commands/secret-init.en.md

```diff
@@ -0,0 +1,94 @@
+# secret init
+```
+$ copilot secret init
+```
+
+## What does it do?
+`copilot secret init` creates or updates secrets as [SecureString parameters](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html#what-is-a-parameter) in SSM Parameter Store for your application.
+
+A secret can have different values in each of your existing environments, and is accessible by your services or jobs from the same application and environment.
+
+!!! attention 
+    Secrets are not supported for Request-Driven Web Services.
+
+## What are the flags?
+```
+  -a, --app string              Name of the application.
+      --cli-input-yaml string   Optional. A YAML file in which the secret values are specified.
+                                Mutually exclusive with name flag and values flag
```

Comment:
Here and below (name, values):
I'm not sure which is preferable, but it appears that elsewhere in the docs where we have mutually exclusive flags, the wording is like this:
"Mutually exclusive with -n --name and --values"

Reply:
I think if we include "flag"/"flags" we also need "the":
"Mutually exclusive with -n, --name and --values" OR "...with the -n, --name and --values flags"
(sorry I accidentally left out the comma above)

Reply:
updated!